gallery/how-to/publishing-packages/publishing-a-package.md

```diff
@@ -61,6 +61,8 @@ Manifest elements that are most important to fill in for item you publish to the
 - Author - this is a required primary key, and contains the name to be associated with the item (see Authors and Owners, below)
 - Description - this is a required primary key, used to briefly explain what this item does and any requirements for using it
 - ProjectURI - this is a strongly recommended URI field in PSData that provides a link to a Github repo or similar location where you do development on the item
+- Tags - it is a strong recommendation to tag your package based on its compatibility with PSEditions and platforms (see [publishing guidlines](/powershell/gallery/concepts/publishing-guidelines.md#tag-your-package-with-the-compatible-pseditions-and-platforms
```

Comment:
Don't use parentheses around this sentence. Make it a new sentence. Also the closing parenthesis for the hyperlink should be on the same line with the rest of the hyperlink.